src/events/dispatchComponentsChanged.js

```diff
@@ -0,0 +1,23 @@
+import store from '../store';
+
+// Using require syntax because of crypto dependency
+const objectHash = require('object-hash');
```

Comment:
Is there another alternative that wouldn't force you to do this? Or is this the canonical package for this purpose?

Reply:
It's a mixture of older modules not being exposed as es modules as well as rollup not playing nicely with said modules, as well as a rollup config setting that could warrant debugging in the future, but isn't worth the investment now.